gce-scripts/running_script.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Runs pshtt with a 10 second time, "Google-Transparency-Report" as the user
```

Comment:
This cites `"Google-Transparency-Report"` as the user agent, but this may not be intended, and I don't see this string mentioned anywhere else in the PR.

Reply:
Also I think this is meant to be `10 second timeout` rather than `10 second time`?